airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/JdbcSource.java

```diff
@@ -39,7 +39,7 @@ public JdbcSource() {
     super("org.postgresql.Driver", SQLDialect.POSTGRES);
   }
 
-  // no-op for JdbcSource since the config it receives is designed to be use for JDBC.
```

Comment:
what is this changing? is JdbcIntegation a thing? I don't think I see it anywhere else.

Reply:
While working on this, I did introduce an abstraction between Jdbc Destination & Source but went backward on that so this should be reverted back too

(in the end, i didn't modify the JdbcSource side)